src/ufixed.jl

```diff
@@ -47,6 +47,9 @@ convert{U<:UFixed}(::Type{U}, x::Real) = _convert(U, rawtype(U), x)
 _convert{U<:UFixed,T}(::Type{U}, ::Type{T}, x)       = U(round(T, widen1(rawone(U))*x), 0)
 _convert{U<:UFixed  }(::Type{U}, ::Type{UInt128}, x) = U(round(UInt128, rawone(U)*x), 0)
 
+rem{T<:UFixed}(x::T, ::Type{T}) = x
+rem{T<:UFixed}(x::UFixed, ::Type{T}) = reinterpret(T, rem(Integer(round((rawone(T)/rawone(x))*reinterpret(x))), rawtype(T)))
```

Comment:
you could use `round(Integer, x)` directly or `reinterpret(T, round((rawone(T)/rawone(x)) * reinterpret(x)) % rawtype(T))` also since this will do floating point division with `Float64` I would replace `Interger` with `Int64`.

It might be worthwhile thinking about using `div(widenmul(rawone(T), reinterpret(x)), rawone(x))` if it doesn't lose to much precision.